applications/Chat/examples/train_reward_model.py

```diff
@@ -80,7 +80,7 @@ def train(args):
     max_len = args.max_len
 
     if args.model == 'llama':
-        tokenizer = prepare_llama_tokenizer_and_embedding(tokenizer, model)
+        tokenizer = prepare_llama_tokenizer_and_embedding(tokenizer, model.model)
```

Comment:
This is related to strategy. This function modify the embedding layer directly, if model is sharded, this may lead to errors.

Reply:
We have to figure out:
1. Is it resize embedding essential? I found `Llama` model in `transformers` is not required to resize after initialization.
2. If it's essential, should we complete this before `prepare()`?

Reply:
> We have to figure out:
> 
> 1. Is it resize embedding essential? I found `Llama` model in `transformers` is not required to resize after initialization.
> 2. If it's essential, should we complete this before `prepare()`?

I assign to pad_token same as unk_token, and the embedding should not resize.